packages/e2e-tests/specs/editor/various/copy-cut-paste-whole-blocks.test.js

```diff
@@ -63,4 +64,21 @@ describe( 'Multi-block selection', () => {
 		await pressKeyWithModifier( 'primary', 'v' );
 		expect( await getEditedPostContent() ).toMatchSnapshot();
 	} );
+
+	it( 'should respect inline copy in places like input fields and textareas', async () => {
+		await insertBlock( 'Shortcode' );
+		await page.keyboard.type( '[my-shortcode]' );
+		await pressKeyWithModifier( 'shift', 'ArrowLeft' );
+		await pressKeyWithModifier( 'shift', 'ArrowLeft' );
+
+		await pressKeyWithModifier( 'primary', 'c' );
```

Comment:
How does this work? As for as I remember, Puppeteer doesn't support copy/paste events and I had to create it myself, but I didn't create it for `input` elements, only for `contenteditable`.

Reply:
Could this have been fixed in the meantime? I just extended the tests, watched them run in `interactive`, and confirmed that headless results were consistent. Nothing suggested a lack of support.

Could you link to the support you built for copy/paste?

Reply:
> Could you link to the support you built for copy/paste?

Never mind, I see it: https://github.com/WordPress/gutenberg/blob/4857ad58c1241b3d63d21a6880c989b85746c3dc/packages/e2e-test-utils/src/press-key-with-modifier.js#L84

Reply:
@ellatrix: but overall are you good with the PR?